core/Translation/Translator.php

```diff
@@ -114,7 +114,8 @@ public function setCurrentLanguage($language)
      */
     public function getDefaultLanguage()
     {
-        return Config::getInstance()->General['default_language'];
+        $generalSection = Config::getInstance()->General;
+        return @$generalSection['default_language'] ?: 'en';
```

Comment:
Could you add it in a comment too? that way we have clues if we want to try to refactor it later